src/pid_design.jl

```diff
@@ -302,3 +302,32 @@ function loopshapingPI(P,ωp; ϕl=0,rl=0, phasemargin = 0, doplot = false)
     end
     return kp,ki,C
 end
+
+"""
+    kp, ki, C = placePI(P, ω₀, ζ; series=false)
+
+Selects the parameters of a PI-controller such that the poles are placed
+to match the poles of s^2 + 2ζω₀ + ω₀^2.
+
+Default is to return the parameters for the parallel form, but if `series=true`
+they will be returned on series form.
+"""
+function placePI(P, ω₀, ζ; series=false)
+    num = numpoly(P)[].coeffs
```

Comment:
```suggestion
    num = numvec(P)[]
```
but this might give coefficients in the reverse order

Reply:
Haha, thought we should have something like that... Had all my variables in switched order first.